mmv1/products/bigtable/AppProfile.yaml

```diff
@@ -141,3 +156,17 @@ properties:
         description: |
           If true, CheckAndMutateRow and ReadModifyWriteRow requests are allowed by this app profile.
           It is unsafe to send these requests to the same table/row/column in multiple clusters.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'standardIsolation'
+    default_from_api: true
+    description: |
+      The standard options used for isolating this app profile's traffic from other use cases.
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: 'priority'
```

Comment:
Lets mark this as required so a user can't set an empty block for standard_isolation {}

Reply:
is this required? if not, I think this is supported behavior, because our API will use a default for the priority if it isn't specified

Reply:
required: true on a sub-field only makes it required if the parent field is specified.

So in this case, a user can not specify `standard_isolation` and that's fine, but if they do specify `standard_isolation` then they must specify priority, which is what we want.

Reply:
I think I understand what you mean - right now it would make sense to have `required: true`, since `standard_isolation` only has 1 field so users can't have an empty block

we're planning to add more fields to `standard_isolation` in the future and we're okay with a user choosing not to specify a `priority` if they're only interested in the other fields (that we'll add in the future), so I think if we add `required: true` now, we'll just end up removing that in the future

lmk if I should still add `required: true` - I'm fine with it but just wanted to give some more context

Reply:
Yeah, we should still add required: true. In the future we can replace it with at_least_one_of that includes all fields